data/wacom.stylus

```diff
@@ -298,63 +310,91 @@ Axes=Tilt;Pressure;Distance;RotationZ;
 Type=Marker
 
 [0x56a:0x100804]
+# Mangled tool ID for kernel < 6.20
```

Comment:
is it going to be in 6.20? Right now we're in the 6.18 cycle, so there's a bit of prediction going on? :)